src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -1913,3 +1913,18 @@ void IslandWindow::UseMica(const bool newValue, const double /*titlebarOpacity*/
     const int attribute = newValue ? DWMSBT_MAINWINDOW : DWMSBT_NONE;
     std::ignore = DwmSetWindowAttribute(GetHandle(), DWMWA_SYSTEMBACKDROP_TYPE, &attribute, sizeof(attribute));
 }
+
+// Method Description:
+// - This method is called when the window receives the WM_NCCREATE message.
+// Return Value:
+// - The value returned from the window proc.
+[[nodiscard]] LRESULT IslandWindow::OnNcCreate(WPARAM wParam, LPARAM lParam) noexcept
+{
+    const auto ret = BaseWindow::OnNcCreate(wParam, lParam);
+    if (!ret)
+    {
+        return FALSE;
+    }
+
+    return TRUE;
+}

```

Comment:
Wait, we don't need _either_ `xxxIslandWindow::OnNcCreate` now? they are both no-ops...

Reply:
to be fair, i think we should still use TrySetDarkTheme in NCCREATE because otherwise we risk a white titlebar flash before it goes dark. NCCREATE is so early, it's to our benefit to do it there.

Reply:
That's... a better point. Okay yea I'll bring that back.